contrib/jmx-metrics/src/test/resources/otlp_config.properties

```diff
@@ -1,4 +1,4 @@
-otel.jmx.interval.milliseconds = 3000
+otel.jmx.interval.milliseconds = 5000
```

Comment:
Is this a polling interval?  3s is already pretty slow...was this increase absolutely necessary?

Reply:
No, but it was helpful w/ https://github.com/open-telemetry/opentelemetry-java-contrib/pull/31#discussion_r617864459.  Reverting.